### PR TITLE
update vcpkg action version

### PR DESCRIPTION
**Pull request by smk762**
To test:
- please download binaries from https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/4596910754
- Launch the portable / installer versions for each OS you can (note which OS you confirmed in review comments).
- Basic smoke test of app (login, do swap)

### CODE REVIEW

**Review by cipig — APPROVED** [review]
works fine under debian 11

**Review by endrilickollari — DISMISSED** [review]
Looks very good on windows 10.

File: .github/workflows/atomicdex-desktop-ci.yml [hunk 1]
```diff
@@ -181,11 +193,11 @@ jobs:
           echo "CC=clang-12" >> $GITHUB_ENV
 
       - name: vcpkg deps (All)
-        uses: lukka/run-vcpkg@v10
+        uses: lukka/run-vcpkg@v11
         with:
           vcpkgDirectory: '${{ github.workspace }}/ci_tools_atomic_dex/vcpkg-repo'
-          appendedCacheKey: ${{ hashFiles('vcpkg.json') }}
           vcpkgJsonGlob: 'vcpkg.json'
+          appendedCacheKey: vcpkginstall
```
Comment (cipig) [inline .github/workflows/atomicdex-desktop-ci.yml:200]: saw this just now:
```
Warning: Unexpected input(s) 'appendedCacheKey', valid inputs are ['vcpkgDirectory', 'runVcpkgInstall', 'vcpkgGitCommitId', 'vcpkgGitURL', 'doNotUpdateVcpkg', 'doNotCache', 'vcpkgJsonGlob', 'vcpkgJsonIgnores', 'binaryCachePath', 'runVcpkgFormatString', 'useShell', 'logCollectionRegExps']
```
i have no idea what it does but seems like `runVcpkgInstall` would be a "correct" value

**Review by Canialon — DISMISSED** [review]
Looks good on MacOS 13 and Windows 11

File: .github/workflows/atomicdex-desktop-ci.yml (hunk 1 above)
Comment (smk762) [reply .github/workflows/atomicdex-desktop-ci.yml:200]: the way caching works changed, so the old key is obsolete. `runVcpkgInstall` should remain false (as the default)

**Pull request merged**